fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -636,9 +642,38 @@ protected void replaceResourceWithStream(final FedoraResource resource,
             throw new RepositoryRuntimeException(e);
         }
 
+        ensureConsistencyOfInput(inputModel);
+
         resource.replaceProperties(translator(), inputModel, resourceTriples);
     }
 
+    private void ensureConsistencyOfInput(final Model inputModel) throws BadRequestException {
+        LOGGER.info("ensuring consistency!");
```

Comment:
Can you trim down this logging? I think `info` is too much here, and we probably want only `debug` if there is no exception.